chainntnfs/txnotifier.go

```diff
@@ -873,7 +873,7 @@ func (n *TxNotifier) dispatchConfDetails(
 	ntfn *ConfNtfn, details *TxConfirmation) error {
 
 	// If no details are provided, return early as we can't dispatch.
-	if details == nil {
+	if details == nil || ntfn.dispatched {
```

Comment:
Nit: we shouldn't log the message below when `details != nil && ntfn.dispatched`.

Reply:
Done